ui/narrative/methods/align_reads_using_hisat2/display.yaml

```diff
@@ -114,7 +117,7 @@ parameters :
 description : |
     <p>This App aligns the sequencing reads for a set of two or more samples to long reference sequences of a genome using HISAT2 and outputs a set of alignments for the given sample set or reads set in BAM format. If the user has only a single sample of reads, this App will take single-end or paired-end library object instead of a sample set.</p>
 
-    <p>In addition, it outputs the alignment statistics such as total reads, mapped and unmapped reads, singletons, multiple alignments, and alignment rate in the table format.</p>
+    <p>In addition, it outputs the Qualimap generated BAM alignment QC report that includes global summary such as total number of mapped reads, mean samples coverage, mean samples GC-content, mean samples mapping quality, and sample statistics for each sample along with PCA and coverage statistics in histograms.</p>
```

Comment:
Some unclear grammar here, and seems to be a run-on sentence. I suggest:

"In addition, it outputs the Qualimap-generated BAM alignment QC report with a global summary. This summary includes the total number of mapped reads, mean samples coverage, mean samples GC-content, mean samples mapping quality, and sample statistics for each sample, along with PCA and coverage histograms."